scripts/release/npm-publish.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
If you're not using [semantic-release](http://semantic-release), you might check out [np](http://npm.im/np) 👍